dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/content/ContentMap.java

```diff
@@ -61,16 +64,18 @@
 	private Structure structure;
 	private String title;
 	private Context context;
-
+    public ContentMap(Contentlet content, User user, PageMode mode, Host host, Context context) {
+        this( content,  user, !mode.showLive,  host,  context) ;
+    }
 	public ContentMap(Contentlet content, User user, boolean EDIT_OR_PREVIEW_MODE, Host host, Context context) {
-		this.content = content;
-		this.conAPI = APILocator.getContentletAPI();
-		this.perAPI = APILocator.getPermissionAPI();
-		this.fields = FieldsCache.getFieldsByStructureInode(content.getStructureInode());
-		this.user = user;
-		this.EDIT_OR_PREVIEW_MODE = EDIT_OR_PREVIEW_MODE;
-		this.host = host;
-		this.context = context;
+        this.content = content;
+        this.conAPI = APILocator.getContentletAPI();
+        this.perAPI = APILocator.getPermissionAPI();
+        this.fields = FieldsCache.getFieldsByStructureInode(content.getStructureInode());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getStructureInode"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)